Latest/Model/AppLibrary.swift

```diff
@@ -47,20 +47,43 @@ class AppLibrary {
 	@objc func gatherAppBundles() {
 		self.appSearchQuery.disableUpdates()
 		
-		// Run metadata query to gather all apps
-		self.bundles = self.appSearchQuery.results.compactMap { item -> App.Bundle? in
-			guard let metadata = item as? NSMetadataItem,
-				let path = metadata.value(forAttribute: NSMetadataItemPathKey) as? String else { return nil }
-			
-			let url = URL(fileURLWithPath: path)
-			
-			// Only allow apps in the application folder and outside excluded subfolders
-			if !Self.applicationPaths.contains(where: { url.path.hasPrefix($0) }) || Self.excludedSubfolders.contains(where: { url.path.contains($0) }) {
-				return nil
+		let appSearchQueryResults = self.appSearchQuery.results
+		if appSearchQueryResults.isEmpty {
+			// Look for applications in the /Applications folder manually
+			let fileManager = FileManager.default
+			var bundles = [App.Bundle]()
+			do {
+				for applicationPath in Self.applicationPaths {
+					let appPaths = try fileManager.contentsOfDirectory(atPath: applicationPath)
+					for appPath in appPaths {
+						let url = URL(fileURLWithPath: applicationPath + "/" + appPath)
```

Comment:
Hint: There is an API for adding path components. 
I would write it like so: 
`URL(fileURLWithPath: applicationPath).appendingPathComponent(appPath)`

Reply:
Done